webstart.ts

```diff
@@ -1,14 +1,17 @@
 import {BasicREPL} from './repl';
 import { Type, Value } from './ast';
 import { defaultTypeEnv } from './type-check';
-import { NUM, BOOL, NONE } from './utils';
+import {NUM, BOOL, NONE, STRING} from './utils';
 
 function stringify(typ: Type, arg: any) : string {
   switch(typ.tag) {
     case "number":
       return (arg as number).toString();
     case "bool":
       return (arg as boolean)? "True" : "False";
+    // convert from ASCII to Character
+    case "str":
+      return String.fromCharCode(arg as number);
```

Comment:
That's true, because we call this method every time we load a ascii code from the memory. The result will be returned and recorded in the same  "pre" lable. Then we can show the whole string.